docs/tutorials/AddRemark.md

```diff
@@ -1,37 +1,34 @@
 ---
-layout: page title: "Tutorial: Adding a command"
+layout: page
+title: "Tutorial: Adding a command"
 ---
```

Comment:
Please revert this file as it is mostly unchanged. Actually, do we even need this?

Reply:
Resolved in #181